fusor-ember-cli/app/mixins/configure-environment-mixin.js

```diff
@@ -19,7 +19,7 @@ export default Ember.Mixin.create(NeedsDeploymentMixin, {
 
   priorLibraryEnvironments: Ember.computed.filter('lifecycleEnvironments', function(item) {
     return (item.get('prior_id') === 1);
-  }).property('lifecycleEnvironments.[]', 'libraryEnv'),
+  }),
```

Comment:
shouldn't have dependent keys on `property()` when using syntx `Ember.computed.filter`. surprised that this worked before commit bc16ff4496ec31107222325d29f759b33212e1a2 caused it to break.

merging.